scripts/pre-push

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Can we also add a script that can be executed by `prow` to ensure docs are generated and there is no diff? We will need to configure that [here](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/osdctl/openshift-osdctl-master-presubmits.yaml) after.